CHANGELOG.rst

```diff
@@ -7,7 +7,7 @@ Changelog for Mantarray File Manager
 - Added file migration function to go from v0.3.1 to v0.4.1 and v0.4.1 to v0.4.2
 - Bumped H5 File version to 0.4.2
 - Added metadata for original file version from Mantarray, file version prior to migration, and time migration was performed
-- Added method to trim h5 files by indicating the amount of centimilliseconds to be trimmed from the start and end.
+- Added function 'h5_file_trimmer' to trim h5 files by indicating the amount of centimilliseconds to be trimmed from the start and end.
```

Comment:
flanking with double backticks `` is what makes it look like code. it will look like this:


``h5_file_trimmer``